test/test_jit.py

```diff
@@ -1583,6 +1583,93 @@ def foo(x):
         test_x = torch.rand(6, 3)
         self.assertEqual(foo(test_x), traced(test_x))
 
+    def test_export_dropout(self):
+        class DropoutTest(torch.nn.Module):
+            def __init__(self):
+                super(DropoutTest, self).__init__()
+                self.dropout = torch.nn.Dropout()
+
+            def forward(self, x):
+                return self.dropout(x)
+
+        test = DropoutTest()
+        test.eval()
```

Comment:
Testing dropout export in eval mode doesn't seem to be the best idea, because an identity graph would pass this test too. Can't we use `torch.random.fork_rng`?

Reply:
I am also just ok keep the test as is. In some sense, I just wanted to make sure networks in eval mode with dropout would export correctly as a smoke test that we didn't exclude a ton of models. This was because previously Dropout was a special python operator.